### PR TITLE
[Website] Publications

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Features publications page and updates to Solutions pages
- Features publications backend schema implementation

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?
- View publications page through deploy preview

#### What are the relevant tickets?
- [WEB-340]

#### Screenshots
image

image
 
 

[WEB-340]: https://airqoteam.atlassian.net/browse/WEB-340?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/backend/publications/models.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from author.decorators import with_author
+from django.db import models
+from cloudinary.models import CloudinaryField
+from backend.utils.models import BaseModel
+
+@with_author
```
Comment (Baalmart) [inline website/backend/publications/models.py:6]: thanks @FaithDaka for this.

File: website/backend/publications/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (Baalmart) [inline website/backend/publications/tests.py:1]: in the near future, we shall all be required to have unit tests for all our implementations. Glad that we already have it in consideration

**Review by Baalmart — APPROVED** [review]
Thanks @FaithDaka , this LGTM. Good stuff!

From an accessibility standpoint, I was wondering how easy it is for our visitors to reach some  of this new content. I do not expect an answer from you now but it would be nice for us to review it in due course.

Otherwise, I will perform merge duty after @Codebmk reviews/approval.

Thanks again!

**Review by Codebmk — COMMENTED** [review]
Code LGTM. Small issue:
Some publications are missing links
Screenshot from 2023-01-06 13-30-50

**Comment by FaithDaka** [comment]
No links were added @Codebmk

**Review by Codebmk — APPROVED** [review]
LGTM Thanks @FaithDaka

**Pull request merged**